shifthelper/logging.py

```diff
@@ -39,16 +40,26 @@ def config_logging(to_console=False, level=logging.DEBUG):
     ))
     formatter.converter = time.gmtime  # use utc in log
     logfile_handler.setFormatter(formatter)
+    stream_handler = logging.StreamHandler(sys.stdout)
+    stream_handler.setLevel(level)
+    formatter = logging.Formatter(
+        '%(asctime)s - %(levelname)s - %(name)s - %(funcName)s - %(message)s'
+    )
+    formatter.converter = time.gmtime  # use utc in log
+    stream_handler.setFormatter(formatter)
+
+    for logger_name in ('shifthelper', 'custos'):
+        log = logging.getLogger(logger_name)
+        log.setLevel(level)
+        log.addHandler(logfile_handler)
+        log.addHandler(text_logfile_handler)
+
+        if to_console:
+            log.addHandler(stream_handler)
 
-    log = logging.getLogger('shifthelper')
-    log.setLevel(level)
+    log = logging.getLogger('root')
```

Comment:
I think this is wrong. The way to get the root logger is: `getLogger()` or `getLogger('')`, 
```
In [5]: logging.getLogger('') == logging.getLogger()
Out[5]: True

In [6]: logging.getLogger('') == logging.getLogger('root')
Out[6]: False
```

Or did you actually want the logger named `root`, If so I didn't know we had a logger of that name.